input/pagecontent/general-requirements.md

```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+
+Systems **MAY** populate and accept other code systems but this is not a requirement of AU Core.
+
+### Missing Data
+
+There are situations when information for a  data element is missing and the  system does not know reason for the absence of data. 
+
+When an element definition is *optional* (minimum cardinality = 0), including elements labelled *Must Support*, an
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is not known.  
+
+When element definition is *Mandatory* (minimum cardinality > 0), 
+- AU Core Responder **SHALL** correctly populate the element *even when* the system does not know a value or the reason a the value is absence. 
+
+The core specification provides guidance for what to do in this situation, which is summarised below.
+
+1.  For *non-coded* data elements including type [Reference](http://hl7.org/fhir/R4/references.html#Reference), 
+  - when the AU Core profile for the element does not require a sub-element
+    - use the [DataAbsentReason extension](http://hl7.org/fhir/R4/extension-data-absent-reason.html) in the data type 
+    - use the code `unknown` - The value is expected to exist but is not known.
+  
+      Example: ExplanationOfBenefit resource where the patient's insurance coverage is not available.
+      ~~~
+      {
+        "resourceType" : "ExplanationOfBenefit",
+            ...
+            "outcome" : "complete",
+              "insurance" : [
+                {
+                  "focal" : true,
+                  "coverage" : {
+                    "extension" : [
+                      {
+                        "url" : "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
+                        "valueCode" : "unknown"
+                      }
+                    ]
+                  }
+                }
+              ],
+              ...
+          }
+      ~~~
+      
+  - when the AU Core profile mandates a sub-element, such as a valid Identifier or Reference data type, then the resource must contain the sub-element otherwise the resource will not be conformant,
+
+1. For *coded* data elements:
+    - *required* binding strength (CodeableConcept or code datatypes):
+      - the appropriate "unknown" concept code **SHALL** be populated if available.
+      - For AU Core profiles, the following *mandatory* or *conditionally mandatory* status elements with required binding have no appropriate "unknown" concept code:
+        - `AllergyIntolerance.clinicalStatus`
+        - `Condition.clinicalStatus`
+        - `Immunization.status`
+    - *example*, *preferred*, or *extensible* binding strengths (CodeableConcept, or Coding datatypes):
+      - when the system has text but no coded value, only the text sub-element is populated.
```

Comment:
missing conformance verb

Reply:
removed conformance verbs from this section instead